exdb/models.py

```diff
@@ -136,6 +136,42 @@ def get_url(self, user):
             return reverse('view_experience', args=[self.pk])
         return reverse('edit', args=[self.pk])
 
+    def get_csv_row(self):
+        row = []
+        row.append(self.name)
+        row.append(self.get_status_display())
+        row.append(str(self.author))
+        planners = ''
+        for planner in self.planners.all():
+            planners += str(planner) + " "
```

Comment:
This results in an extra space at the end of the string. Alternately, you could do:

```python
planners = ', '.join([str(p) for p in self.planners.all()])
```